src/scripts/list_pipelines.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A script to detect all pipelines declared in pipelines/ and cache/ and
+# dump a summary of their configurations within a `tmp/` folder in the
+# project root.
+
```

Comment:
Can you add an example command line and mention what directory the script should be run from?

Reply:
Yes, thanks.